its/ruling/src/test/resources/expected/python-S1244.json

```diff
@@ -0,0 +1,186 @@
+{
+"project:biopython/Bio/PDB/internal_coords.py": [
+3219
+],
+"project:django-2.2.3/django/contrib/gis/db/models/fields.py": [
+248
+],
+"project:django-2.2.3/django/contrib/gis/db/models/functions.py": [
+420
+],
+"project:docker-compose-1.24.1/tests/unit/timeparse_test.py": [
+8,
+12,
+16,
+20,
+28
+],
+"project:mypy-0.782/test-data/stdlib-samples/3.2/random.py": [
+518
+],
+"project:numpy-1.16.4/numpy/core/function_base.py": [
+408,
+408
+],
+"project:numpy-1.16.4/numpy/core/tests/test_multiarray.py": [
+4464,
+4464,
+4877,
+4886,
+7489
+],
+"project:numpy-1.16.4/numpy/core/tests/test_numeric.py": [
+99
+],
+"project:numpy-1.16.4/numpy/core/tests/test_records.py": [
+108,
+109
+],
+"project:numpy-1.16.4/numpy/core/tests/test_scalarmath.py": [
+237,
+238
+],
+"project:numpy-1.16.4/numpy/core/tests/test_ufunc.py": [
+518,
+521
+],
+"project:numpy-1.16.4/numpy/core/tests/test_umath.py": [
+231,
+326,
+327,
+924,
+925,
+982,
+983
+],
+"project:numpy-1.16.4/numpy/f2py/crackfortran.py": [
+2157,
+2157
+],
+"project:numpy-1.16.4/numpy/lib/function_base.py": [
+420,
```

Comment:
There are quite a few issues raised. I am not sure about all  the `np.sum(a == 0.0)` it seems like a TP for me.